src/shadertoymanager.ts

```diff
@@ -111,6 +111,13 @@ export class ShaderToyManager {
         }
     }
 
+    public postCommand = (command : string) => {
+        if (this.webviewPanel !== undefined) {
+            this.webviewPanel.Panel.webview.postMessage({command: command});
+        }
+        this.staticWebviews.map((webview: StaticWebview) => webview.Panel.webview.postMessage({command: command}));
```

Comment:
Array.prototype.map() implies that you're creating another array. So either use a plain for...of loop or use Array.prototype.forEach()

Reply:
Or would it be possible to post only to the currently focused webview? If that's not trivial I can investigate in the future.

Reply:
> Array.prototype.map() implies that you're creating another array. So either use a plain for...of loop or use Array.prototype.forEach()

Also a good point. Thanks for the suggestion.

> Or would it be possible to post only to the currently focused webview? If that's not trivial I can investigate in the future.

In brief, yes. This issue could be further explored. It will be quite easy to do if you are focusing on one of the webviews instead of the text editor (I suppose that they cannot be focused on at the same time?). But I also want to be able to send the commands when focusing on text editor, which can work in a way like how you select the webview to update in `shadertoyManager.onDocumentChanged()`. The problem is, if one document can have zero or multiple previews, do you save/pause all of them? I found such design still somehow counter-intuitive, so I just sent the command to all for now.

Reply:
Right, I can see that it is worth it to have these commands working when a text editor is focused. I don't necessarily see a problem with select all valid previews based on an editor and posting the commands to those. But I think that's something we can tackle in the future, preserving the behaviour you have implemented with an setting.